parachain/primitives/core/src/outbound.rs

```diff
@@ -278,6 +278,14 @@ pub trait SendMessage {
 	fn deliver(ticket: Self::Ticket) -> Result<H256, SendError>;
 }
 
+/// A trait for getting the local costs associated with sending a message.
+pub trait OutboundQueueLocalFee {
```

Comment:
To match the naming the `SendMessage` trait.

```suggestion
pub trait SendMessageFee {
```

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/999/commits/be36fbe3a8f0b9f9478a900a479d5e59e8fdd12d